lib/utils/logger.ts

```diff
@@ -104,19 +105,20 @@ export class Logger {
 		this._alertLevel = level;
 	}
 
+	/**
+	 * Add a list of strings to the list of strings to redact.
+	 * @param secrets  Secrets to redact from any output.
+	 */
+	public pushSecrets(secrets: string[]): void {
```

Comment:
However, considering the comment above, arguably this should be a setter:
```typescript
public set secrets(...) { ... }
```
In which case, the class member name *should* be `_secrets`. :)